setup.py

```diff
@@ -28,6 +28,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/wpilibsuite/sphinxext-opengraph",
+    license="LICENSE.md",
```

Comment:
Isn't this inconsistent with renaming from LICENSE.md to LICENSE

Reply:
Indeed, it's a leftover from an earlier version (it's the first time I have to create a python package)